modules/apps/portal-security/portal-security-antisamy/build.gradle

```diff
@@ -465,5 +465,26 @@ private String _filterKnowledgeBaseSanitizer(String line) {
 		|	</tag>""".stripMargin()
 	}
 
+	if (line.contains('<regexp name="systemColor" value="(activeborder|activecaption')) {
+		line = """\
+			|${line}\n
```

Comment:
Hey @thienquach87 
Do we have a reason for having new line character here? Also #line477
I just did a quick search in the file, all other occurrences of ${line} are followed by a new line though.
Also in line#483, you don't have new line.

Reply:
line#470 => make [space](https://github.com/liferay-appsec/liferay-portal/blob/ec4838617df45d14678cf5cda674087625d6de9c/modules/apps/portal-security/portal-security-antisamy/src/main/resources/META-INF/resources/knowledge-base-sanitizer-configuration.xml#L117) between 2 lines(2 component)
line 477=> was move to 478 to make [space](https://github.com/liferay-appsec/liferay-portal/blob/ec4838617df45d14678cf5cda674087625d6de9c/modules/apps/portal-security/portal-security-antisamy/src/main/resources/META-INF/resources/knowledge-base-sanitizer-configuration.xml#L752) between 2 lines(2 component) 
line 483, [it](https://github.com/liferay-appsec/liferay-portal/blob/ec4838617df45d14678cf5cda674087625d6de9c/modules/apps/portal-security/portal-security-antisamy/src/main/resources/META-INF/resources/knowledge-base-sanitizer-configuration.xml#L374-L376) is in the same component so we don't need new line character